blueprints/lazylibrarian/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/local/bin/bash
+# This file contains the install script for lazylibrarian
+
```

Comment:
Please add the following to be compatible with the dev branch.

```suggestion

#init jail
initblueprint "$1"

```